ffi/src/main/java/com/antithesis/ffi/internal/FfiHandler.java

```diff
@@ -38,14 +38,12 @@ public long initializeModuleCoverage(long edgeCount, String symbolFilePath) {
             throw new IllegalArgumentException("Antithesis Java instrumentation supports [1 ," + Integer.MAX_VALUE + "] edges");
         }
         offset = FfiWrapperJNI.init_coverage_module(edgeCount, symbolFilePath);
-        String msg = String.format("Initialized Java module at offset 0x%016x with %d edges; symbol file %s", offset, edgeCount, symbolFilePath);
```

Comment:
We should output this `'msg'` - it might be load bearing, or could become so, and seems generally informative.  Just don't need to get involved with a logger to show it.